capi/README.md

```diff
@@ -1,3 +0,0 @@
-# capi cookbook
```

Comment:
Why do you want to delete our README.md @petergtz? It contains so much useful info.